include/errors.h

```diff
@@ -188,601 +125,22 @@ enum generic_identifiers {
 # define SWO_IOL_BLE_0C                   (ERR_IOL_BLE + ERR_GEN_ID_0C)    // 0x150C
 #endif // HAVE_BLE
 
-/**
- * Tasks-related issues are categorized into:
- * Stack issues (STC),
- * Task Owner issues (OWN),
- * Overflow issues (OFW).
- */
-#define ERR_TSK_STC                       (ERR_TSK_RANGE + ERR_GEN_SUB_01)
-#define ERR_TSK_OWN                       (ERR_TSK_RANGE + ERR_GEN_SUB_02)
-#define ERR_TSK_OFW                       (ERR_TSK_RANGE + ERR_GEN_SUB_03)
-
-#define SWO_TSK_STC_01                    (ERR_TSK_STC + ERR_GEN_ID_01)    // 0x2101
-#define SWO_TSK_STC_02                    (ERR_TSK_STC + ERR_GEN_ID_02)    // 0x2102
-#define SWO_TSK_STC_03                    (ERR_TSK_STC + ERR_GEN_ID_03)    // 0x2103
-#define SWO_TSK_STC_04                    (ERR_TSK_STC + ERR_GEN_ID_04)    // 0x2104
-#define SWO_TSK_STC_05                    (ERR_TSK_STC + ERR_GEN_ID_05)    // 0x2105
-#define SWO_TSK_STC_06                    (ERR_TSK_STC + ERR_GEN_ID_06)    // 0x2106
-#define SWO_TSK_STC_07                    (ERR_TSK_STC + ERR_GEN_ID_07)    // 0x2107
-#define SWO_TSK_STC_08                    (ERR_TSK_STC + ERR_GEN_ID_08)    // 0x2108
-
-#define SWO_TSK_OWN_01                    (ERR_TSK_OWN + ERR_GEN_ID_01)    // 0x2201
-#define SWO_TSK_OWN_02                    (ERR_TSK_OWN + ERR_GEN_ID_02)    // 0x2202
-#define SWO_TSK_OWN_03                    (ERR_TSK_OWN + ERR_GEN_ID_03)    // 0x2203
-#define SWO_TSK_OWN_04                    (ERR_TSK_OWN + ERR_GEN_ID_04)    // 0x2204
-
-#define SWO_TSK_OFW_01                    (ERR_TSK_OFW + ERR_GEN_ID_01)    // 0x2301
-#define SWO_TSK_OFW_02                    (ERR_TSK_OFW + ERR_GEN_ID_02)    // 0x2302
-
-/**
- * Cryptography parameter-related issues are categorized into:
- * Lengths mismatch in functions (LEN),
- * Values mismatch in functions (VAL).
- */
-#define ERR_CRY_LEN                       (ERR_CRY_RANGE + ERR_GEN_SUB_01)
-#define ERR_CRY_VAL                       (ERR_CRY_RANGE + ERR_GEN_SUB_02)
+#define ERR_APP_APD_CLA                   (ERR_APP_APD_RANGE + ERR_GEN_SUB_0E) // 0x6E00
+#define ERR_APP_APD_INS                   (ERR_APP_APD_RANGE + ERR_GEN_SUB_0D) // 0x6D00
+#define ERR_APP_APD_HDR                   (ERR_APP_APD_RANGE + ERR_GEN_SUB_05) // 0x6500
+#define ERR_APP_APD_STA                   (ERR_APP_APD_RANGE + ERR_GEN_SUB_06) // 0x6600
+#define ERR_APP_APD_LEN                   (ERR_APP_APD_RANGE + ERR_GEN_SUB_07) // 0x6700
+#define ERR_APP_APD_DAT                   (ERR_APP_APD_RANGE + ERR_GEN_SUB_08) // 0x6800
```

Comment:
Why do you need to add _APP in the name?
This introduce a breaking change that doesn't seems necessary? Though this doesn't break many apps it seems.

Reply:
`ERR_APD_RANGE` is used to define the OS error codes. Keeping the name makes a redefinition which is not allowed.